climada/util/config.py

```diff
@@ -213,11 +220,18 @@ def dir(self, index=None, create=True):
         Exception
             if the value is not a string or if the directory cannot be created
         """
-        path = Path(self.str(index)).expanduser()
+        path = self._expand_source_dir(Path(self.str(index)).expanduser())
         if create:
             path.mkdir(parents=True, exist_ok=True)
         return path.absolute()
 
+    @staticmethod
+    def _expand_source_dir(path):
+        parts = path.parts
+        if parts[0] == '...':
+            return Path(Config.SOURCE_DIR, *parts[1:])
+        return Path(*parts)
```

Comment:
Since this function accesses the class, wouldn't it be more natural to have a class method instead of a static method?
```suggestion
    @classmethod
    def _expand_source_dir(cls, path):
        parts = path.parts
        if parts[0] == '...':
            return Path(cls.SOURCE_DIR, *parts[1:])
        return Path(*parts)
```

Reply:
Agreed. Cleaner that way. (Even though `Config` is not meant to be extended)